Asgardia.Constitution.Proposal.txt

```diff
@@ -59,19 +59,19 @@ To ensure the existence and development of Asgardia and humanity’s expansion t
 Article 4. Core Values of Asgardia
 § 1. Asgardia’s Core Values are the common and permanent foundations of the self-identification of Asgardian citizens. These Core Values form the commonality of Asgardian society and lie at the heart of the objectives and activities of Asgardian state bodies, citizens, and associations.
 § 2. The Constitution, laws, and other regulations of Asgardia, as well as the activities of its state bodies, implement and protect Asgardia’s Core Values.
-§ 3. Asgardia’s absolute Prime Directive is humanity reaching out into the Universe, explore and seek out new life, and go where no one has gone before.
+§ 3. Asgardia’s Prime Directive is for humanity to reach out into the Universe, explore and seek out new life, and go where no one has gone before.
 § 4. Asgardia’s Core Values in fulfilling its Mission are:
  a. peace in space and the peaceful settlement of the Universe;
  b. protection of all of humanity from space-originating threats;
  c. ensuring equality of opportunity in space for all Asgardian citizens;
- d. unity of humanity as a community in space;
- e. human dignity, human rights, freedoms, and progressive development of the individual;
- f. supremacy of law; 
- g. space ecology;
- h. community and mutual support; 
- i. scientific advancement, artistic creativity, and the belief in the boundless potential of human reason, knowledge, labour, and progress;
- j. harmonious existence of the individual, society, and the Space Nation.
-§ 5. Asgardia’s Core Values are equally important and equally implemented in state administration. In the event of a conflict between the Core Values, the absolute Prime Directive of Asgardia prevails.
+ d. the unity of humanity as a community in space;
+ e. protecting human dignity, human rights, freedoms, and progressive development of the individual;

```

Comment:
I prefer to use "defend" over "protect." Seems that _protect_ can be a bit forceful.